### PR TITLE
ME: form field wrapper, abstract record field

### PR DESCRIPTION
**Pull request by LHBruneton-C2C**
### Description

This PR introduces a new component to share the logic of adding a title and help tooltip to each record form field.

The form field for abstract is now specifically using the rich form field, based on markdown-editor and the new wrapper.

### Architectural changes

Form fields have been move from ui-inputs to feature-editor.

### Screenshots

image

### Quality Assurance Checklist

- [X] Commit history is devoid of any _merge commits_ and readable to facilitate reviews
- [X] If **new logic** ⚙️ is introduced: unit tests were added
- [ ] If **new user stories** 🤏 are introduced: E2E tests were added
- [X] If **new UI components** 🕹️ are introduced: corresponding stories in Storybook were created
- [ ] If **breaking changes** 🪚 are introduced: add the `breaking change` label
- [ ] If **bugs** 🐞 are fixed: add the `backport <release branch>` label
- [ ] The [documentation website](docs) 📚 has received the love it deserves

### CODE REVIEW

File: libs/feature/editor/src/lib/components/record-form/form-field/form-field.component.stories.ts
Comment (jahow) [inline libs/feature/editor/src/lib/components/record-form/form-field/form-field.component.stories.ts:1]: These stories do not work anymore; maybe they're not relevant now?

**Review by jahow — APPROVED** [review]
Really nice refactoring, thanks! it makes a lot of sense to me.

When opening the editor I noticed that the rich text field wasn't adjusting its height correctly in preview mode, see:

image

Also the storybook stories for the `form-field` component are broken now.

I think we can merge this PR and revisit some small things about the display (size, icons) later to not make this PR too complex. I'm leaving it to you to decide what to do in this PR and what to do later :slightly_smiling_face:

**Comment by LHBruneton-C2C** [comment]
@jahow Thanks for the review!
I don't know if you've noticed, but the datafeeder is not building in production anymore, complaining about markdown-parser CSS being too large (I didn't change anything there, but it might be "affected").
Do you have any suggestion to fix or bypass this error?

**Comment by jahow** [comment]
> I don't know if you've noticed, but the datafeeder is not building in production anymore, complaining about markdown-parser CSS being too large (I didn't change anything there, but it might be "affected").
> Do you have any suggestion to fix or bypass this error?

It might be an issue with bundle size budgets? This value should be bumped a bit: https://github.com/geonetwork/geonetwork-ui/blob/179fba8309222a1dc00319e80504123785b231a5/apps/datafeeder/project.json#L44

**Comment by LHBruneton-C2C** [comment]
TODO: fix markdown-parser height.

**Pull request merged**